app/models/chargeSummary/ChargeSummaryViewModel.scala

```diff
@@ -66,11 +66,11 @@ case class ChargeSummaryViewModel(
     chargeItem.hasLpiWithDunningLock || (paymentBreakdown.nonEmpty && hasDunningLocks) || (paymentBreakdown.nonEmpty && hasInterestLocks)
   }
 
-  val taxYearFrom = chargeItem.taxYear - 1
-  val taxYearTo = chargeItem.taxYear
+  val taxYearFrom = chargeItem.taxYear.endYear - 1
```

Comment:
you can use one of the shortcuts 
chargeItem.taxYear.endYear => chargeItem.taxYear.startYear  or chargeItem.taxYear.previousYear.endYear